tests/test_jira_api.py

```diff
@@ -20,7 +20,7 @@ def test_login_to_jira(self, response, expected):
     @allure.title("Post issue")
     @pytest.mark.api
     def test_post_issue(self):
-        response = api.post_issue("OA API Issue001", "OlegAverkin", "Low")
+        response = api.post_issue("OA API Issue", "OlegAverkin", "Low")
         assert response.status_code == 201
```

Comment:
Please, also check they type of content.
Also validate returned data, confirm that it contains required text.

Reply:
Done